tools/cldr-apps/js/src/esm/cldrMenu.js

```diff
@@ -90,7 +90,22 @@ function loadInitialMenusFromJson(json) {
   addTopLocale("root", theDiv);
   for (let n in locmap.locmap.topLocales) {
     const topLoc = locmap.locmap.topLocales[n];
-    addTopLocale(topLoc, theDiv);
+    const topLocInfo = locmap.getLocaleInfo(topLoc);
+    if (topLocInfo.special_type !== "scratch") {
+      // Skip Sandbox locales here
+      addTopLocale(topLoc, theDiv);
+    }
+  }
+  if (cldrStatus.getIsUnofficial()) {
+    $(theDiv).append($("<hr/><h4>Test Locales</h4>"));
+    for (let n in locmap.locmap.topLocales) {
+      const topLoc = locmap.locmap.topLocales[n];
+      const topLocInfo = locmap.getLocaleInfo(topLoc);
+      if (topLocInfo.special_type === "scratch") {
+        // Now only Sandbox locales here
+        addTopLocale(topLoc, theDiv);
+      }
+    }
   }
```

Comment:
This function needs subroutines -- adding 15 lines makes it longer than it should be

Reply:
@btangmu that could be good, but I think even better would be modernizing the locale list sidebar.